server/routes/server.routes.js

```diff
@@ -34,7 +34,7 @@ router.get('/:username/sketches/:project_id/add-to-collection', (req, res) => {
 });
 
 router.get('/:username/sketches/:project_id', (req, res) => {
-  projectForUserExists(req.params.username, req.params.project_id, exists => (
+  projectForUserExists(req.params.username, req.user, req.params.project_id, exists => (
```

Comment:
`projectForUserExists` is called a few times in the application, have the other times that it's called been updated to work with the new parameters?

Reply:
I am not sure, how should I check that?

Reply:
What I would do is search all files for instances where this function is being called. I do this in VSCode by hitting <kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>F</kbd>, then typing in the function name `projectForUserExists`, and then it should list all of the instances!

Reply:
I realized `projectForUserExists` function definition does not include the parameter that handles the `req.user` argument that's being passed on line 37. Not sure what happened to the code that was written to handle it